print.py

```diff
@@ -1 +1,2 @@
 print('Hello world')
+print('Addition check')
```

Comment:
Multiplication instead of addition?
```suggestion
print('Multiply check')
```